.github/workflows/package.yaml

```
@@ -209,7 +209,7 @@ jobs:
     - name: Verify installer signature
       shell: powershell
       run: |
-        $usedCert = (Get-AuthenticodeSignature -FilePath 'dist\Rancher Desktop Setup*.msi').SignerCertificate
+        $usedCert = (Get-AuthenticodeSignature -FilePath 'dist\Rancher*Desktop*.msi').SignerCertificate
```

Not sure why this doesn't use a `.` in the first position, for consistency, but it doesn't really matter.

```suggestion
        $usedCert = (Get-AuthenticodeSignature -FilePath 'dist\Rancher.Desktop*.msi').SignerCertificate
```

---

I opted to use `*` so we don't have to worry about it as much.

---

Yeah, but just above you use `path: dist/Rancher.Desktop*.msi`, so you still have to worry about it anyways. That's why I wrote "for consistency". Still doesn't really matter though. 😄